Update README.adoc to clarify behaviour of stream-logs (#37)
hazelcast/hazelcast-cloud-maven-plugin: README.adoc

----

|stream-logs
|(experimental) Stream the latest 1,000 lines of logs from a single member in the connected cluster.
|

> |If a cluster member is restarted for any reason, logs stream will be interrupted and needs to be restarted as well.

Reviewer: ```suggestion
|If a cluster member is restarted for any reason, logs stream will be interrupted and must be restarted.
```
